server/start-dev.js

```diff
@@ -1,13 +1,23 @@
 'use strict';
 
+console.log('start-dev process.env.NODE_ENV', process.env.NODE_ENV);
+
 const path = require('path');
 const nodemon = require('nodemon');
 const { build } = require('vite');
 const mergeOptions = require('merge-options');
 
 const viteConfig = require('../vite.config');
 
-console.log('start-dev process.env.NODE_ENV', process.env.NODE_ENV);
+// Build the client-side JavaScript bundle when we see any changes
+build(
+  mergeOptions(viteConfig, {
+    build: {
+      // Rebuild when we see changes
+      watch: true,
+    },
+  })
+);
```

Comment:
Just putting the Vite build above so it's close to the `viteConfig` that we import